app/Http/Controllers/TalksController.php

```diff
@@ -102,7 +85,7 @@ public function store()
     public function edit($talkId)
     {
         try {
-            $talk = Auth::user()->talks()->findOrFail($talkId);
+            $talk = Auth::user()->talks()->withTrashed()->findOrFail($talkId);
```

Comment:
Hey Matt, sounds great. Just to clarify, do you mean for example adding a boolean column called archived, then just making a scope to reference all the articles where that is true?